gooddata-java/src/main/java/com/gooddata/sdk/service/GoodData.java

```diff
@@ -331,19 +182,17 @@ public void logout() {
      *
      * @return initialized service for project management
      */
-    @Bean("goodDataProjectService")
```

Comment:
I would vote for keeping bean definitions in this "convenience" helper class - those changes will then be transparent even for spring bean users.

Reply:
done